### DISCUSSION TITLE
How to draw outside the canvas?

### DISCUSSION THREAD

**Discussion by Quikkby** [keiwando/toonsquid#41] [Q&A] [answered]
my friend is having a problem with the canvas where he cannot draw outside

one solution he found is making the canvas bigger manually with transform tool (with edit pixels off)

but is there a easier way?  its annoying for him and me to do that

**Comment by keiwando** [comment, marked as answer]
There is currently no option to resize or crop the canvas after a project has been created, but this will be added in the future. 

By the way, when "edit pixels" is disabled, you are just scaling up the pixel layers with the transform tool. The resolution of the layer stays the same as before, so you're not actually gaining any extra space to draw on.

If you're just trying to for example add like annotations around the canvas, you can already to that with the vector brushes. You could also create a higher-resolution project and then add a camera and scale it down, which would give you some extra space around the camera to draw on with pixel brushes as well.
